_posts/2017-08-21-elasticsearch-loader-0.10.0-released/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: "Elasticsearch Loader 0.10.0 released"
+title-short: Elasticsearch Loader 0.10.0
+tags: [elasticsearch, kinesis, aws, nsq]
+author: Enes
+category: Releases
+---
```

Comment:
there is a permalink now cf https://github.com/snowplow/snowplowanalytics.com/pull/268/files#diff-045d4559e49dfa4323be406263f343cbR8